README.md

```diff
@@ -2,10 +2,14 @@
 
 **`rustvmm/dev`** is a container with all dependencies used for running
 `rust-vmm` integration and performance tests. The container is available on
-Docker Hub and has support for `x86_64` and `aarch64` platforms.
+Docker Hub and has support for `x86_64`, `aarch64`, and `riscv64` platforms.
 
-For the latest available tag, please check the `rustvmm/dev` builds available
-on [Docker Hub](https://hub.docker.com/r/rustvmm/dev/tags).
+For the latest available tag for `x86_64` and `aarch64`, 
+please check the `rustvmm/dev` builds available on 
+[Docker Hub](https://hub.docker.com/r/rustvmm/dev/tags).
+For the latest available tag for `riscv64`, 
+please check the `rustvmm/dev_riscv64` builds available on 
```

Comment:
As I've already commented, it's not clear to me why we have to use a different tag just for `riscv64`. If it's really necessary, I think we should explain it here as well.

Reply:
Resolved by adding more detailed explanation in README.md